lib/agenda/index.js

```diff
@@ -8,13 +8,15 @@ const Emitter = require('events').EventEmitter;
 const humanInterval = require('human-interval');
 
 class Agenda extends Emitter {
-  constructor(config = {}, cb) {
+  constructor(config, cb) {
     super();
 
     if (!(this instanceof Agenda)) {
       return new Agenda(config);
     }
 
+    config = config ? config : {};
```

Comment:
Node v4 and v5 are a pain. [Only 6+ supports default params](http://node.green/#ES2015-syntax-default-function-parameters).